File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        Map<String,Integer> freq = new HashMap<>();
        for(String s:s1.split(" ")){
            freq.putIfAbsent(s,0);
            freq.put(s,freq.get(s)+1);
        }
        for(String s:s2.split(" ")){
            freq.putIfAbsent(s,0);
            freq.put(s,freq.get(s)+1);
        }
        List<String> res = new ArrayList<>();
        for(Map.Entry<String,Integer> e: freq.entrySet()){
            if (e.getValue()==1){
                res.add(e.getKey());
            }
        }
        
        return res.toArray(new String[0]);
    }
}

File: 920-uncommon-words-from-two-sentences/uncommon-words-from-two-sentences.py
class Solution:
    def uncommonFromSentences(self, s1: str, s2: str) -> List[str]:
        freq = dict()
        for w in s1.split():
            if w not in freq:
                freq[w] = 0
            freq[w] += 1
        for w in s2.split():
            if w not in freq:
                freq[w] = 0
            freq[w] += 1
        res = []
        for key,value in freq.items():
            if value == 1:
                res.append(key)
        return res